unleash-yggdrasil/src/lib.rs

```diff
@@ -135,13 +137,21 @@ pub fn compile_state(state: &ClientFeatures) -> HashMap<String, CompiledToggle>
                 compiled_strategy: compile_rule(rule.as_str()).unwrap(),
                 impression_data: toggle.impression_data.unwrap_or_default(),
                 project: toggle.project.clone().unwrap_or("default".to_string()),
+                dependencies: compile_dependencies(&toggle.dependencies),
```

Comment:
This is probably overkill, this can just be

Then we also don't need the `compile_dependencies` function

```suggestion
                dependencies: toggle.dependencies.clone().unwrap_or_default(),
```

Reply:
Yup; good call.

Reply:
handled in 5d58cb6857178c5669a3b6f948243c4c9120196d